core/tests/protractor_utils/forms.js

```diff
@@ -194,7 +194,10 @@ var RichTextEditor = async function(elem) {
   var closeRteComponentButtonLocator = by.css(
     '.e2e-test-close-rich-text-component-editor');
   // Set focus in the RTE.
-  await waitFor.elementToBeClickable(rteElements.first());
+  await waitFor.elementToBeClickable(
+    rteElements.first(),
+    'First RTE element'
+  );
```

Comment:
This part of the code is giving [this ](https://github.com/oppia/oppia/runs/7777125375?check_suite_focus=true#step:13:579) error now. PTAL. Also, the error message doesn't seen to be descriptive. It should be something like `First RTE element is not clickable`.

Ditto in the webdriverio code.

Reply:
done